pkg/operator/baremetal_pod.go

```diff
@@ -65,42 +116,125 @@ func setMariadbPassword() corev1.EnvVar {
 	}
 }
 
-func generateRandomPassword() string {
-	rand.Seed(time.Now().UnixNano())
+func setIronicHtpasswdHash(name string, secretName string) corev1.EnvVar {
+	return corev1.EnvVar{
```

Comment:
If we're not already setting a password in an environment variable in a deployment spec, I don't think we should start now.  This data needs to go into a secret and the application configured to consume that secret.

Reply:
It's not a password, it's a bcrypt hash of a password for the server to use to verify connections.